### PR TITLE
Remove portable- targetgroups

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/corefx/issues/31321

@ericstj this is currently failing when building the System.ValueTuple.pkgproj with the following error:

```
C:\git\corefx\Tools\Packaging.targets(1126,5): error : System.ValueTuple should support API version 4.0.4.0 on .NETFramework,Version=v4.5 but C:\git\corefx\packages\System.ValueTuple\4.4.0\ref\portable-net40+sl4+win8+wp8\System.ValueTuple.dll was found to support 4.0.2.0. [C:\git\corefx\src\System.ValueTuple\pkg\System.ValueTuple.pkgproj]
C:\git\corefx\Tools\Packaging.targets(1126,5): error : System.ValueTuple has a higher runtime version (4.0.4.0) than compile version (4.0.2.0) on .NET Desktop framework .NETFramework,Version=v4.5.  This will break bindingRedirects.  If the live reference was replaced with a harvested reference you may need to set <Preserve>true</Preserve> on your reference assembly ProjectReference. [C:\git\corefx\src\System.ValueTuple\pkg\System.ValueTuple.pkgproj]
```

The description is already quite explanatory but I'm still confused. Can you please explain what is necessary here to make it work?

### CODE REVIEW

**Comment by ViktorHofer** [comment]
cc @weshaggard do you know what is required here?

**Comment by ericstj** [comment]
@ViktorHofer the first error is because you left the <SupportedFramework>net45</SupportedFramework> on the ref, but the one that was applying for net45 is now harvested (and a different version than the live build).  So you should remove that, or move it to the harvest element if you like the way that looks, though it is redundant since we make sure all frameworks supported by the harvested package are supported by the live one by default.

The latter issue may just be fallout, but you should check the package report to understand what's going on if it isn't clear.  You can find the package report in the output (bin\packages\<configuration>\reports).  It'll show you which asset applies everywhere and what its version is.  That'll help you understand why net45 might be getting mismatched ref/lib.

File: src/System.ValueTuple/pkg/System.ValueTuple.pkgproj [hunk 1]
```diff
@@ -10,6 +10,8 @@
     <InboxOnTargetFramework Include="netcoreapp2.0" />
     <InboxOnTargetFramework Include="uap10.0.16299" />
     <InboxOnTargetFramework Include="$(AllXamarinFrameworks)" />
+
+    <HarvestIncludePaths Include="ref/portable-net40+sl4+win8+wp8;lib/portable-net40+sl4+win8+wp8" />
```
Comment (weshaggard) [inline src/System.ValueTuple/pkg/System.ValueTuple.pkgproj]: @ericstj knows better but I think you want to add SupportedFramework=net45 here and bump the SupportedFramework to net461 in the ProjectReference for the ref above. I think that should get you a step closer to figuring out the validation error.

File: src/System.ValueTuple/pkg/System.ValueTuple.pkgproj [hunk 2]
```diff
@@ -3,13 +3,18 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
```
Comment (weshaggard) [inline src/System.ValueTuple/pkg/System.ValueTuple.pkgproj]: @ericstj and I were talking about this today and we think we can completely remove this package now.

**Pull request merged**